nexus/src/db/model.rs

```diff
@@ -339,10 +327,8 @@ impl From<Generation> for sled_agent_client::types::Generation {
 /// Representation of a [`u16`] in the database.
 /// We need this because the database does not support unsigned types.
 /// This handles converting from the database's INT4 to the actual u16.
-#[derive(
-    Copy, Clone, Debug, Eq, Ord, PartialEq, PartialOrd, AsExpression, FromSqlRow,
-)]
-#[sql_type = "sql_types::Int4"]
```

Comment:
curious if `AsExpression` was also superfluous before or something changed to make it so

Reply:
It may have been used on a trait bound - I don't see an explicit call to `as_expression`.